privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -626,11 +626,11 @@ def getUserId(self, LoginName):
                             LoginName))
 
         for entry in r:
-            userid = self._get_uid(entry, self.uidtype)
+            userid = str(self._get_uid(entry, self.uidtype))
```

Comment:
Also, what do you think about moving the stringification into ``_get_uid``?

Reply:
Done in 61743ea